hermes/src/api/types.rs

```diff
@@ -179,3 +189,81 @@ impl RpcPriceIdentifier {
         RpcPriceIdentifier(id.to_bytes())
     }
 }
+
+#[derive(Clone, Copy, Debug, serde::Deserialize, serde::Serialize)]
+pub enum EncodingType {
+    #[serde(rename = "base64")]
+    Base64,
+    #[serde(rename = "hex")]
+    Hex,
+}
+
+impl Default for EncodingType {
+    fn default() -> Self {
+        EncodingType::Hex
+    }
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
+pub struct BinaryPriceUpdate {
+    pub encoding: EncodingType,
+    pub data:     Vec<String>,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
+pub struct ParsedPriceUpdate {
+    pub id:        String,
+    pub price:     RpcPrice,
+    pub ema_price: RpcPrice,
+    pub metadata:  RpcPriceFeedMetadataV2,
+}
+
+#[derive(Debug, Clone, serde::Serialize, serde::Deserialize, ToSchema)]
+pub struct PriceUpdate {
+    pub binary: BinaryPriceUpdate,
+    #[serde(skip_serializing_if = "Option::is_none")]
+    pub parsed: Option<Vec<ParsedPriceUpdate>>,
+}
+
+impl PriceUpdate {
+    pub fn from_price_feed_update(
```

Comment:
Where is this used?

Reply:
removed to use `ParsedPriceUpdate::from_price_feed_update`